content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -146,7 +161,7 @@ automatically create spans for each incoming request.
 
 ```shell
 npm install @opentelemetry/sdk-node \
-  @opentelemetry/api \
+  @opentelemetry/sdk-trace-node \
```

Comment:
I remember that **some** versions of Node.JS required the `api` to be here explicitly.

I am wondering if we can skip sdk-trace-node and sdk-metrics alltogether since sdk-node has both as dependency:

https://www.npmjs.com/package/@opentelemetry/sdk-node?activeTab=dependencies

@open-telemetry/javascript-approvers PTAL!

Reply:
Noted @svrnm . I believe the sdk-trace-node and sdk-metrics could be dropped if we get the imports directly from sdk-node but it would require some extra destructuring in order to get the classes that are being used

Reply:
> but it would require some extra destructuring in order to get the classes that are being used

not sure if I understand what you mean by that?

Reply:
makes sense, thx